node_to_python.py

```diff
@@ -462,14 +462,62 @@ def create_main():
         file.close()
         return {'FINISHED'}
 
+class NodeToPythonMenu(bpy.types.Menu):
+    bl_idname = "NodeToPythonMenu"
```

Comment:
style: [bpy documentation](https://docs.blender.org/api/current/bpy.types.Menu.html) recommends something like "NODE_MT_node_to_python" for the `bl_idname`

Reply:
would need to replace reference in line 507 as well